UTNAK/kepler16b-using-imce-vocabulary
# Feat/20 demonstration of jupyter notebook as an analysis engine

## Checklist before submitting a merge request


## Description of contribution

I have built a Jupyter notebook that has a similar function to analysisShell.

## Testing performed


## How to Test Expected functionality changes

1. Open 'src/analysis/analysisShell.ipynb'
2. On terminal, send commands
    - conda init
    - CLOSE shells
    - conda activate py39
3. From command pallet, `python select interpreter` -> `py39
    - sometimes, py39 doesn't appear. please click reload of command pallet
4. In jupyter notebook, Select Python Kernel to appropriate environment in this simulation. (normally, py39 created by .sh)
5. Using OML Vision or terminal, Run Build and Run Load ( .sh also include these commands, no need to run normally)
6. Following the script in Jupyter Notebook, run all cells.
7. Using OML Vision or terminal, Run Save
8.  Verify the oml description file `src/oml/example.com/tutorial2/description/statedictionary.oml` is updated.

<img width="709" alt="image" src="https://github.com/UTNAK/kepler16b-using-imce-vocabulary/assets/29029659/296c2499-b872-49da-8e71-ddeaf6b41a62">


## Additional context

[If needed, you may add additional context]
